app/services/otp_rate_limiter.rb

```diff
@@ -15,19 +15,14 @@ def exceeded_otp_send_limit?
   end
 
   def max_requests_reached?
-    return throttle.throttled? if IdentityConfig.store.redis_throttle_otp_rate_limiter_read_enabled
-
-    entry_for_current_phone.otp_send_count > otp_maxretry_times
+    return throttle.throttled?
   end
 
   def rate_limit_period_expired?
-    return throttle.expired? if IdentityConfig.store.redis_throttle_otp_rate_limiter_read_enabled
-    otp_last_sent_at.present? && (otp_last_sent_at + otp_findtime) < Time.zone.now
+    return throttle.expired?
```

Comment:
Same:

```suggestion
    throttle.expired?
```

Reply:
Alternatively, do we need these methods at all anymore, or could we just call the throttle method directly?

Reply:
I'd like to consider it, yeah. The `OtpRateLimiter` interface is used in multiple places though, so I'm not sure.